<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSubjectDialog</class>
 <widget class="QDialog" name="AddSubjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 221, 218);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Subject</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>511</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>subject</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subject_name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_subject_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 212, 129);
color:white;
border:none;
border-radius: 8px;
font-weight:bold;
font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Add subject</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(88, 88, 88);
color:white;
border:none;
border-radius: 8px;
font-weight:bold;
font-size: 15px;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="description_field">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
